graf3d/eve7/src/REveElement.cxx

```diff
@@ -601,7 +601,7 @@ TClass *REveElement::IsA() const
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Export render-element to CINT with variable name var_name.
+/// Export render-element to Cling with variable name var_name.
```

Comment:
Can we maybe spin off all the eve changes into a separate PR? This code is not owned by us folks at CERN, to it would be better to keep the changes for a different PR reviewed by @alja

Reply:
Sure, done. :)